cloudflare.php

```diff
@@ -468,6 +469,16 @@ function purgeCache()
     }
 }
 
+function sslRewrite() {
+    if ( isset( $_SERVER['HTTP_X_FORWARDED_PROTO'] ) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https' ) {
+        $_SERVER['HTTPS'] = 'on';
+
+        return true;
+    }
+
+    return false;
```

Comment:
I know.  We'll figure that out at some point.  I only mention it to point out that we should be writing all our code in a way that is easy to test even if we can't actually write a test for it.